src/index.css

```diff
@@ -18,20 +18,27 @@ a {
   color: #646cff;
   text-decoration: inherit;
 }
+
 a:hover {
   color: #535bf2;
 }
 
 body {
   margin: 0;
-  display: flex;
+  width: 100%;
   place-items: center;
   min-width: 320px;
   min-height: 100vh;
+  justify-content: center;
+  width: 100%;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  background-color: lavender
```

Comment:
Zatím to schválím i takto, ale na příště:
- width tady je 2x
- nastavovat width: 100% je většinou zbytečné a často (a kontraproduktivní)
- justify-content je flexbox vlastnost